flutter/linux/sentry_flutter_plugin.cc

```diff
@@ -0,0 +1,62 @@
+#include "include/sentry_flutter/sentry_flutter_plugin.h"
+
+#include <flutter_linux/flutter_linux.h>
+#include <gtk/gtk.h>
+#include <sys/utsname.h>
+
+#include <cstring>
+
+#define SENTRY_FLUTTER_PLUGIN(obj) \
+  (G_TYPE_CHECK_INSTANCE_CAST((obj), sentry_flutter_plugin_get_type(), \
+                              SentryFlutterPlugin))
+
+struct _SentryFlutterPlugin {
+  GObject parent_instance;
+};
+
+G_DEFINE_TYPE(SentryFlutterPlugin, sentry_flutter_plugin, g_object_get_type())
+
+// Called when a method call is received from Flutter.
+static void sentry_flutter_plugin_handle_method_call(
+    SentryFlutterPlugin* self,
+    FlMethodCall* method_call) {
+  g_autoptr(FlMethodResponse) response = nullptr;
+
+  const gchar* method = fl_method_call_get_name(method_call);
+
+  response = FL_METHOD_RESPONSE(fl_method_not_implemented_response_new());
```

Comment:
I think we can add support for https://github.com/getsentry/sentry-native here.

Reply:
yep, let's not do it, for now, let's just be sure that this plugin does nothing instead of throwing unless it never gets called.

Reply:
it never get's called at all.

Reply:
It's great this uses CMake so it should be easy to add sentry-native later.